SPECS/python-pytest-xdist/python-pytest-xdist.spec

```diff
@@ -65,7 +65,7 @@ pip3 install atomicwrites>=1.3.0 \
     apipkg
 PATH=%{buildroot}%{_bindir}:${PATH} \
 PYTHONPATH=%{buildroot}%{python3_sitelib} \
-    python%{python3_version} -m pytest -v  -k "not test_warning_captured_deprecated_in_pytest_6"
+    python%{python3_version} -m pytest -v  -k "not test_warning_captured_deprecated_in_pytest_6 and not test_fixture_teardown_failure"
```

Comment:
Can we please document with a comment as to why are we skipping the `fixture_teardown` test